server/events.lua

```diff
@@ -168,18 +168,16 @@ RegisterNetEvent('QBCore:Server:UseItem', function(item)
     QBCore.Functions.UseItem(src, item)
 end)
 
-RegisterNetEvent('QBCore:Server:RemoveItem', function(itemName, amount, slot)
+-- This event is exploitable and should not be used. Will be deprecated soon. function(itemName, amount, slot)
+RegisterNetEvent('QBCore:Server:RemoveItem', function(itemName, amount, _)
     local src = source
-    local Player = QBCore.Functions.GetPlayer(src)
-    if not Player then return end
-    Player.Functions.RemoveItem(itemName, amount, slot)
+    print(string.format("%s triggered QBCore:Server:RemoveItem by ID %s for %s %s. This event is being deprecated because of exploitation. Adjust your events accordingly to do this server side with player functions.", GetInvokingResource(), src, amount, itemName))
 end)
 
-RegisterNetEvent('QBCore:Server:AddItem', function(itemName, amount, slot, info)
+-- This event is exploitable and should not be used. It has been deprecated. function(itemName, amount, slot, info)
+RegisterNetEvent('QBCore:Server:AddItem', function(itemName, amount, _, _)
     local src = source
-    local Player = QBCore.Functions.GetPlayer(src)
-    if not Player then return end
-    Player.Functions.AddItem(itemName, amount, slot, info)
+    print(string.format("%s triggered QBCore:Server:AddItem by ID %s for %s %s. This event is being deprecated because of exploitation. Adjust your events accordingly to do this server side with player functions.", GetInvokingResource(), src, amount, itemName))
```

Comment:
```suggestion
    print(string.format("%s triggered QBCore:Server:AddItem by ID %s for %s %s. This event is deprecated due to exploitation. Adjust your events accordingly to do this server side with player functions.", GetInvokingResource(), src, amount, itemName))
```
English can be improved here